setup.cfg

```diff
@@ -21,6 +21,7 @@ install_requires =
     astropy
     asdf>=2.8.0
     numpy
+    asdf-coordinates-schemas @ git+https://github.com/eslavich/asdf-coordinates-schemas.git@eslavich-create-package
```

Comment:
I assume this will be updated after https://github.com/asdf-format/asdf-coordinates-schemas/pull/1 is merged and package released.

Reply:
Yes!  And it done!